### ISSUE TITLE
Track-Changes Plugin

### ISSUE THREAD

**Issue by aris-konstantinidis** [leapt/froala-editor-bundle#18]
While trying to use the `track-changes` plugin, I came across the following issues:
- The `PluginProvider` does not containt this plugin in the `ARR_PLUGIN_CONFIG` list.
- After adding it, the plugin functionality is still not showing up in the editor. After some debugging, I found out that the problem lies in the `froala_widget.html.twig` file, in the part where the key-value option pairs are set. Changing the code in the following way resolves the issue. Any ideas why? Sorry, I did not have the time to dig in deeper in the issue.
```php
{% for option, value in froala_arrOption %}
      {% if option != 'pluginsEnabled' %}
          {{ option }} : {% include "@LeaptFroalaEditor/Form/value.html.twig" %}
      {% endif %}
{% endfor %}
```

- The options `trackChangesEnabled` and `showChangesEnabled` are missing from `UConfiguration.php` in the `OPTIONS_BOOLEAN` list. 

Thank you very much for your awesome package!

**Comment by jmsche** [comment]
@aris-konstantinidis PR welcome :)

**Comment by jmsche** [comment]
Hello @aris-konstantinidis, it has been fixed in v1.6.0 release :)
